#!/usr/bin/env python
PACKAGE = "moving_object_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("cluster_size", int_t, 0, "Minimum number of cluster element", 5, 1, 100)
gen.add("dynamic_speed", double_t, 0, "Velocity length[meter] used to distinguish dynamic points from static points", 0.30, 0.01, 1.0)
gen.add("neighbor_distance", double_t, 0, "Maximum distance[meter] between two points treated as same cluster", 0.1, 0.01, 1.0)

exit(gen.generate(PACKAGE, "moving_object_detector", "EuclideanClusterer"))
